src/app/ui/NewTaskForm.tsx

```diff
@@ -254,6 +254,12 @@ export const NewTaskForm = ({
                     fontSize: '12px',
                     maxWidth: { xs: '60px', sm: '100px' },
                   }}
+                  title={
+                    tempAssignee
+                      ? (tempAssignee as IAssigneeCombined)?.name ||
+                        `${(tempAssignee as IAssigneeCombined)?.givenName ?? ''} ${(tempAssignee as IAssigneeCombined)?.familyName ?? ''}`.trim()
+                      : 'Assignee'
+                  }
```

Comment:
I see this similar logic being repeated in multiple places. Could you make a util for this (getSelectedAssignee already implements this partially so you could extend it without affecting other places it has been used)